src/ListGroup.js

```diff
@@ -21,16 +26,25 @@ class ListGroup extends React.Component {
       });
     }
 
-    if (shouldRenderDiv) {
-      return this.renderDiv(items);
-    }
-    return this.renderUL(items);
+    return shouldRenderDiv ? this.renderDiv(items) : this.renderUL(items);
   }
 
   isAnchorOrButton(props) {
     return (props.href || props.onClick);
   }
 
+  areCustomChildren(children) {
+    let customChildren = false;
+
+    ValidComponentChildren.forEach(children, (child) => {
+      if (child.type.displayName !== 'ListGroupItem') {
```

Comment:
I'm not sure of another way to do this check. Using `child.type` returns the function definition. I agree it is awkward, and if there is a different way to do it, it would probably be preferable.